src/react/useAtom.ts

```diff
@@ -8,7 +8,7 @@ import type {
 import { useAtomValue } from './useAtomValue.ts'
 import { useSetAtom } from './useSetAtom.ts'
 
-type SetAtom<Args extends any[], Result> = (...args: Args) => Result
+type SetAtom<Args extends unknown[], Result> = (...args: Args) => Result
```

Comment:
I thought the `unknwon` was more stable than the `any` 
https://devblogs.microsoft.com/typescript/announcing-typescript-3-0-rc-2/#the-unknown-type

Reply:
Also, `Args extends unknown[]` is already used in many places. 🤗